### ISSUE TITLE
Plugin broek [ENDWALKER]

### ISSUE THREAD

**Issue by Grammernatzi** [Grammernatzi/XIVComboPluginExpandedest#21]
Yeah, it goes without saying nothing's going to be working for Endwalker for a good while. Daemitus is mostly going to be the one working on fixing things up in that regard, as this is simply an addon to his plugin, which is a rewrite of the original plugin. Some features are probably going to be removed, like PLD's combos turning into spells (because of requiescat lasting a long time now and spells no longer interrupting combos). For now, enjoy launch, this'll come back eventually.

**Comment by Grammernatzi** [comment]
Well, I updated it to the point that it should work, but now I can't even log in to test it; so, if anyone else wants to try it out, I pushed the latest version onto github and you can try to build it yourself, if you know how. If you don't, you're probably better off waiting.

**Comment by Grammernatzi** [comment]
Sadly, no, it's still not working; I'll have to wait for Daemitus to get on it when he wakes up

**Comment by Grammernatzi** [comment]
Also, it's worth noting that I won't push a build until Daemitus does; this was at the request of a certain someone. Once I am given the go, I will release, but not until then.

**Comment by shdwp** [comment]
Heartfelt thank you for the very timely updates from the ones that can compile it ourselves!

**Comment by shdwp** [comment]
I think `BattleChara.StatusList` is bugged on EW, it won't display most of the buffs.

**Comment by TheZynster** [comment]
putzing around with it now that i can finally login. Dragoon basic combos work and even the icons switch for most it seems. only combo i noticed that doesn't work correctly is the Coerthan Torment Combo probably ID changed or combo start changed. Can't remember what it was before but it goes Doom Spike>Sonic Thrust>Coerthan Torment. BOTD into Stardiver is no longer needed as.....BOTD no longer is a skill now. 

Full Thrust Combo still good, Chaos Thrust still good. Those two combo's alone remove like....6-7 skills off the bar for insane hotbar management alone. Jump + Mirage Dive also still works. 

Obviously once I hit 86 those combo's will break due to the inclusion of two replacement skills, but shouldn't be too bad to do as the overall combo's do not change. Dragoon seems to be mostly identical as it was in ShB.

All in all thanks for the new version so i can putz around, test and at least get a good chunk of clearance from my bar for now.

**Comment by idecibel** [comment]
If it's not too demanding of your time, do you think one of you guys who have compiled it manually could give a short write up on how to do it? 

I have reduced mobility in my hands and had to give up on FF14 until I found this plugin (thanks a lot to the creators) waiting for all the stuff to be updated has allowed me to catch up on my netflix shows, but I'm really starting to feel the urge now! Just in case some of you have the time - I figured it wouldn't hurt to ask!

**Comment by Grammernatzi** [comment]
> putzing around with it now that i can finally login. Dragoon basic combos work and even the icons switch for most it seems. only combo i noticed that doesn't work correctly is the Coerthan Torment Combo probably ID changed or combo start changed. Can't remember what it was before but it goes Doom Spike>Sonic Thrust>Coerthan Torment. BOTD into Stardiver is no longer needed as.....BOTD no longer is a skill now.
> 
> Full Thrust Combo still good, Chaos Thrust still good. Those two combo's alone remove like....6-7 skills off the bar for insane hotbar management alone. Jump + Mirage Dive also still works.
> 
> Obviously once I hit 86 those combo's will break due to the inclusion of two replacement skills, but shouldn't be too bad to do as the overall combo's do not change. Dragoon seems to be mostly identical as it was in ShB.
> 
> All in all thanks for the new version so i can putz around, test and at least get a good chunk of clearance from my bar for now.

The combos shouldn't break as I made them future proof. Also, the AoE combo seems to be working fine for me. Are you building off the Endwalkerest branch? The main branch is currently outdated.

**Comment by Grammernatzi** [comment]
> If it's not too demanding of your time, do you think one of you guys who have compiled it manually could give a short write up on how to do it?
> 
> I have reduced mobility in my hands and had to give up on FF14 until I found this plugin (thanks a lot to the creators) waiting for all the stuff to be updated has allowed me to catch up on my netflix shows, but I'm really starting to feel the urge now! Just in case some of you have the time - I figured it wouldn't hurt to ask for this temporary solution until things are up and running again!

If you have questions, you can DM me on Discord, especially if it's disability-related. But I'm not allowed to publish a public build until the original plugin is updated. This is at the request of someone who I'd prefer not to name to keep them from being harassed about it. I don't think they'd want someone to post build instructions here, either.

**Comment by Grammernatzi** [comment]
> I think `BattleChara.StatusList` is bugged on EW, it won't display most of the buffs.

So far buff tracking seems okay. I can't seem to read debuff features on enemies at the moment, but as only one feature used that, it's not too bad. Summoner gauge is also broken, so the vanilla Aetherflow features don't work.

**Comment by shdwp** [comment]
Yeah, self buff tracking has been fixed.

**Comment by idecibel** [comment]
> If you have questions, you can DM me on Discord, especially if it's disability-related (I am on the Dalamud discord server). But I'm not allowed to publish a public build until the original plugin is updated. This is at the request of someone who I'd prefer not to name to keep them from being harassed about it. I don't think they'd want someone to post build instructions here, either.
>

Thanks for the reply and the offer, I might take you up on it eventually, but i'll give it a few more days to respect & not circumvent the agreement you guys have made.

**Comment by Grammernatzi** [comment]
Also, I figure I would give a heads up, but sometimes buff tracking breaks. This is a Dalamud issue at the moment, and is being worked on. I can't do anything about it. It's unfortunate, with the queue times and login errors, but it is what it is.

**Comment by TheZynster** [comment]
> > putzing around with it now that i can finally login. Dragoon basic combos work and even the icons switch for most it seems. only combo i noticed that doesn't work correctly is the Coerthan Torment Combo probably ID changed or combo start changed. Can't remember what it was before but it goes Doom Spike>Sonic Thrust>Coerthan Torment. BOTD into Stardiver is no longer needed as.....BOTD no longer is a skill now.
> > Full Thrust Combo still good, Chaos Thrust still good. Those two combo's alone remove like....6-7 skills off the bar for insane hotbar management alone. Jump + Mirage Dive also still works.
> > Obviously once I hit 86 those combo's will break due to the inclusion of two replacement skills, but shouldn't be too bad to do as the overall combo's do not change. Dragoon seems to be mostly identical as it was in ShB.
> > All in all thanks for the new version so i can putz around, test and at least get a good chunk of clearance from my bar for now.
> 
> The combos shouldn't break as I made them future proof. Also, the AoE combo seems to be working fine for me. Are you building off the Endwalkerest branch? The main branch is currently outdated.

oh, yes im just running the old one. I have no flipping clue how to build the plugin for dalamund lol

**Comment by dvdio** [comment]
> If you have questions, you can DM me on Discord, especially if it's disability-related (I am on the Dalamud discord server). 

What is your discord address?
